seshdash/tests.py

```diff
@@ -1,3 +1,24 @@
 from django.test import TestCase
+from seshdash.models import Sesh_Alert, Alert_Rule, Sesh_Site, BoM_Data_Point as Data_Point
+from seshdash.utils import alert
+from datetime import datetime
 
-# Create your tests here.
+class AlertTestCase(TestCase):
+    site = Sesh_Site.objects.create(site_name=u"Test site", comission_date=datetime(2015, 12, 11, 22, 0),
```

Comment:
Thanks for these tests!

A few comments explaining the tests would be helpful. I know it's pretty trivial right now but it won't be forever.